### PR TITLE
feat(scalability): adding checks for compression and skipped file

### PR DESCRIPTION
**Pull request by joshkurz**
**Description of changes:** 

* Adding in a check in scalability section from https://aws.github.io/aws-eks-best-practices/scalability/docs/control-plane/#disable-kubectl-compression

* Also adding a skipped.json file. Not sure if this is cool, but trying to keep track of what I am skippig for now, and maybe we could go back and print out what was skipped in each section later if a user wanted to see it. IF anything it's good to keep track of the ones we are skipping over.

I have only tested this by running the tests. I did not compile and run this. Tests are passing before i mocked function and after.

### CODE REVIEW

**Comment by joshkurz** [comment]
sent PR to move this to `cluster` https://github.com/aws/aws-eks-best-practices/pull/290. Will update this PR based on the kubeconf docs.

**Comment by joshkurz** [comment]
Updated PR to check clusters and created a more common `get_cluster_config` method and update `hardeneks.init.py` to call the helpers.

All the tests are passing, and my new test is passing. I usually test with and without mocking on a real kube config file. I need to run an e2e test by compiling it. Curious what the best way to do that is. Ive done it before, But I hacked the file to allow me to call it, and i'm wondering if there is a better way.

Also, the changes I made do not allow for `kubeconfig` to be passed in, but it will be a simple addition later on. I just didnt want to change too much as it already got much bigger. From what I can see, there is no option for users to pass in `kubeconfig` and we expect it to be in `~/.kube/config`. I think we can make this the default and allow for users to pass this in as well. envVars still work, but it should be a param as well.

File: hardeneks/cluster_wide/scalability/control_plane.py [hunk 1]
```diff
@@ -22,3 +23,24 @@ def check_EKS_version(resources: Resources):
...
+
+#
+# check_kubectl_compression
+# checks all clusters in config for disable-compression flag set to true
+# if any cluster does not have setting, it returns False
+def check_kubectl_compression(resources: Resources):
+    kubeconfig = helpers.get_kube_config()
+    isSetCorrectly = True
+    for cluster in kubeconfig.get("clusters", []):
+        clusterName = cluster.get("name", "NoName")
```
Comment (dorukozturk) [inline hardeneks/cluster_wide/scalability/control_plane.py:35]: Can we check this for the current cluster only? Resources object should have the cluster name:

Something like:
```python
if clusterName == resources.cluster:
    if cluster.get("cluster", {}).get.......:    
```

File: hardeneks/cluster_wide/scalability/control_plane.py [hunk 2]
```diff
@@ -22,3 +23,24 @@ def check_EKS_version(resources: Resources):
...
+        if cluster.get("cluster", {}).get("disable-compression", False) != True:
+            isSetCorrectly = False
+            console.print(
+                Panel(
+                    f"[red]DisableCompression in Cluster {clusterName}  should equal True",
```
Comment (dorukozturk) [inline hardeneks/cluster_wide/scalability/control_plane.py:40]: Can we change DisableCompression to disable-compression

**Review by dorukozturk — COMMENTED** [review]
Can you also add this new rule to the config.yaml file

**Pull request merged**